### DISCUSSION TITLE
How to disable Replay and Timing text?

### DISCUSSION THREAD

**Discussion by SkyanUltra** [KadeArchive/Kade-Engine#915] [Q&A]
I’m trying to record visuals for an osu! beatmap to play in the background along with some editing, but the timing and replay text get in the way of this and it seems to be hardcoded since it uses that same text in the debug/charting menu. any help with this would be greatly appreciated!

**Comment by isakube** [comment]
heyy it's an osu!fnf charter

uhh in playstate look for `add(replayTxt);` and remove that i think
